### PR TITLE
[ENH, MRG] Add mne.viz.Brain.get_view method

### PR DESCRIPTION
**Pull request by alexrockhill**
Fixes https://github.com/mne-tools/mne-python/issues/10660.

### CODE REVIEW

File: mne/viz/_brain/_brain.py [hunk 1]
```diff
@@ -3046,6 +3046,32 @@ def show(self):
         if self._block:
             _qt_app_exec(self._renderer.figure.store["app"])
 
+    def get_view(self, row=0, col=0):
```
Comment (alexrockhill) [inline mne/viz/_brain/_brain.py:3049]:
```suggestion
    @fill_doc
    def get_view(self, row=0, col=0):
```

File: mne/viz/_brain/_brain.py [hunk 2]
```diff
@@ -3046,6 +3046,33 @@ def show(self):
...
+
+        Returns
+        -------
+        %(roll)s
+        %(distance)s
+        %(azimuth)s
+        %(elevation)s
+        %(focalpoint)s
+        """
+        _validate_type(row, ('int-like',), 'row')
```
Comment (larsoner) [inline mne/viz/_brain/_brain.py:3068]: `row = _ensure_int(row, 'row')` or so is nicer because it actually casts to int

**Review by larsoner — COMMENTED** [review]
Otherwise LGTM, @GuillaumeFavelier feel free to merge if you're happy

File: mne/viz/_brain/_brain.py (hunk 2 above)
Comment (alexrockhill) [inline mne/viz/_brain/_brain.py:3068]:
```suggestion
        row = _ensure_int(row, 'row')
```

File: mne/viz/_brain/_brain.py [hunk 3]
```diff
@@ -3046,6 +3046,33 @@ def show(self):
...
+        _validate_type(col, ('int-like',), 'col')
```
Comment (alexrockhill) [inline mne/viz/_brain/_brain.py:3069]:
```suggestion
        col = _ensure_int(col, 'col')
```

**Review by GuillaumeFavelier — APPROVED** [review]
LGTM :+1:

**Pull request merged**